BankManagment/UnitOfWork/UnitOfWork.cs

```diff
@@ -0,0 +1,47 @@
+﻿using BankManagmentDB.Context;
+using BankManagmentDB.Entity;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace BankManagmentDB
+{
+    public class UnitOfWork : IUnitOfWork
+    {
+        private readonly ApplicationDbContext _context;
+
+        public UnitOfWork(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        private IRepository<AccountType> _accountTypesRepository;
+        public IRepository<AccountType> AccountTypesRepository =>
```

Comment:
register IRepository of each entity into DI and use DI to get IRepository<AccountType>. You can also use Reflection to register IRepository